WordPress/Classes/ViewRelated/Blog/Blogging Prompts/Bloganuary/BloganuaryOverlayViewController.swift

```diff
@@ -75,6 +78,7 @@ class BloganuaryOverlayViewController: UIViewController {
 
         // Set up the close button in the navigation bar.
         let dismissAction = UIAction { [weak self] _ in
+            BloganuaryTracker.trackModalDismissed()
```

Comment:
This function call is missing a parameter, causing the build to fail.

I'm going to modify the code on my end, assuming the local `promptsEnabled` bool is supposed to be passed here, and continue reviewing the rest of the PR.

Reply:
Thanks. This is fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22130/commits/05356b066e8990b6998eac7dd69eb513b369b17a